keps/sig-api-machinery/1281-network-proxy/README.md

```diff
@@ -403,6 +403,24 @@ Beta:
   are spending dialing out.
 - Ensure we have metrics on each existing egress use case.
 
+GA:
+
+- TBD
+- Documentation is GA quality.
+- Protocol simplification: [#462](https://github.com/kubernetes-sigs/apiserver-network-proxy/issues/462)
+- Post-dial instrumentation: [#410](https://github.com/kubernetes-sigs/apiserver-network-proxy/issues/410)
+- Overload protection.
+  * Proxy client use.
+  * Agent connections.
+- Version skew test coverage.
```

Comment:
Definitely agree on this being a GA requirement. Specifically:
- agent / server skew
- client / server skew

Test framework should parameterize the versions being tested, and we can debate which combinations should be covered by CI separately.

Having CI coverage for version skew testing should also be a GA requirement though.